deployment/ansible/roles/ckanext-datajson/tasks/main.yml

```diff
@@ -3,20 +3,16 @@
 - name: Install datajson dependencies
   command: "{{ ckan_virtualenv_path }}/bin/pip install jsonschema pyyaml lepl"
```

Comment:
I was hoping the install would pull these in and a separate step wouldn't be necessary, but no such luck.  It looks like the [requirements](https://github.com/GSA/ckanext-datajson/blob/datagov/requirements.txt)  on the branch we're pointing to now have changed a little, too.

Maybe we could install the extension first then use the requirements file installed with it (at `/usr/lib/ckan/default/src/ckanext-datajson/requirements.txt`) to install requirements?  It's out of order, but I'm not sure that matters, and it would avoid hard-coding a copy of the requirements into the role.